src/Signature.ts

```diff
@@ -85,18 +85,20 @@ export class Signature {
         return this.signature.type;
     }
 
-    /** Verify a signature with a message digest and public key */
-    public verify(digest: BNInput, publicKey: PublicKey, encoding?: string): boolean {
+    /** Verify a signature with a message or hashed message digest and public key */
+    public verify(data: BNInput, publicKey: PublicKey, shouldHash: boolean = true, encoding: string = 'utf8'): boolean {
```

Comment:
`shouldHash` default should be false for backwards compatibility.

Reply:
`eosjs-ecc` `verify` hashes the data by default, hence `shouldhash` is set to true by default.

Reply:
This comment was only related to backwards compatibility if we released this as `21.x.0`